clean_project/utilities.py

```diff
@@ -16,3 +18,20 @@ def load_punctuation():
             punctuation += (unichr(x).encode("utf-8"))
     return punctuation
 
+
+def search(search_word, memory):
```

Comment:
this file shouldn't be a dump of all the attempts you do at doing something. if you manage to do this in the real code, please delete this code and keep only the bits that you actually used for generating test data or whatever and that don't have a place anywhere ekse